### DISCUSSION TITLE
SVG path shaped area import error

### DISCUSSION THREAD

**Discussion by Josseflos** [WebSVG/voronoi#30] [Q&A]
Hi there,

I am trying to import a SVG file in the Voronoi web editor (Chrome), but keep receiving the same error: "only supported import of SVG with a single path on the top level". I am using Inkscape to create a SVG and am making sure that the file is a single path on the top layer, but even when plucking simples shapes from the internet, the error comes up. Does anyone know what could go wrong, or could anyone share a file that works, so I can compare it with other files? 

Thanks in advance!

Jos

**Comment by wassfila** [comment]
as this issue, I suggest to continue it here #31
